lms/djangoapps/commerce/views.py

```diff
@@ -123,3 +128,10 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
         except ApiError as err:
             # The API will handle logging of the error.
             return InternalRequestErrorResponse(err.message)
+
+
+@cache_page(84600)
```

Comment:
Do we really want to cache this so aggressively?  I'd imagine 30 minutes would be about the limits of what we would tolerate if we accidentally added a typo or misconfigured the support email and needed to hotfix :)